website/addons/mendeley/model.py

```diff
@@ -0,0 +1,356 @@
+# -*- coding: utf-8 -*-
+
+import time
+
+import mendeley
+from modularodm import fields
+
+from framework.exceptions import PermissionsError
+
+from website.addons.base import AddonNodeSettingsBase
+from website.addons.base import AddonUserSettingsBase
+from website.oauth.models import ExternalProvider
+from website.util import web_url_for
+
+from website.addons.citations.utils import serialize_account, serialize_folder
+
+from . import settings
+from .api import APISession
+
+class AddonMendeleyUserSettings(AddonUserSettingsBase):
```

Comment:
For consistency with the most recent addons, do not prefix addon models with `Addon`.

Will make this change when I merge.